code/Controllers/SilverStripeNavigatorItem_CMSLink.php

```diff
@@ -16,13 +17,13 @@ public function getHTML()
         return sprintf(
             '<a href="%s">%s</a>',
             $this->record->CMSEditLink(),
-            _t('SilverStripe\\CMS\\Controllers\\ContentController.CMS', 'CMS')
+            _t(ContentController::class . '.CMS', 'CMS')
```

Comment:
Should this not just use `$this->getTitle()` - if not, it should use the same fallback text as `getTitle()` (third param: `Used in navigation. Should be a short label`)

Reply:
I'm not refactoring the class, I'm just migrating it.

Reply:
I think if you're going to change LoC, then they should be changed so they are fully correct, not partly changed.

By slipping these unrelated changes into a migration commit/PR you're risking regressions for no benefit. If we at least take the chance to improve the implementation when slipping these unrelated changes in, then that somewhat justifies the risks that regressions slip through the net.

Reply:
I'll revert this change tomorrow then I guess.

Reply:
Reverted